openlibrary/templates/wikidata_author.html

```diff
@@ -0,0 +1,19 @@
+$def with (wd_entity)
+
+$if wd_entity:
+    <div class="section wikidatabox read-options">
+        <style>
+            .wikidatabox p {
+                margin-bottom: 0;
+            }
+        </style>
+        <h3 style="margin-top: 2px;margin-bottom: 2px;" class="bold black">Quick Info</h3>
+        <div style="margin: 5px;">
+            <p>$wd_entity.description(i18n.get_locale())</p>
+            <span style="font-size: .6em;">Powered by
+                <a href="http://www.wikidata.org/entity/$wd_entity.id">Wikidata
+                    <img src="/static/images/icons/edit.svg">
+                </a>
+            </span>
```

Comment:
This will be kind of annoying to i18n ; I'd make the whole thing a link

```suggestion
            <a href="http://www.wikidata.org/entity/$wd_entity.id" style="font-size: .6em;">$_('Powered by Wikidata')
                <img src="/static/images/icons/edit.svg">
            </a>
```